src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -1774,27 +1847,25 @@ class IrsPlan extends React.Component<
           }
         }
 
-        const nextAdminTileset = country.tilesets[geographicLevel + 1];
+        const adminLayerIds = layerIds.filter(l => l.includes('jurisdiction-fill')).sort();
+        const nextAdminTileset = tilesets[geographicLevel + 1];
```

Comment:
I think we should add some documentation, maybe via a comment here that this logic depends on the jurisdiction objects `geographicLevel` values being consistently correct (I think this may not be the case right now on the OpenSRP server).